### ISSUE TITLE
owncloud 7.0.3 adding "(2)" to every file/folder shared

### ISSUE THREAD

**Issue by yopzolo** [owncloud/core#12740]
### Steps to reproduce
1. user1, user2
2. user1 shares a directory to user2
3. user2 hits the news api (ex:curl -k https://host/owncloud/index.php/apps/news/api/v1-2/version -u user2 -p

### Expected behaviour
nothing except getting the news..

### Actual behaviour
All user2's shares have a (2) appended in theyre name (it can happen multiple times)

### Server configuration
**Operating system**:
Raspbian GNU/Linux 7 (wheezy)

**Web server:**
nginx

**Database:**
mysql

**PHP version:**
PHP 5.4.35-0+deb7u2 (cli) (built: Nov 19 2014 13:13:59) 
Copyright (c) 1997-2014 The PHP Group
Zend Engine v2.4.0, Copyright (c) 1998-2014 Zend Technologies
    with XCache v2.0.0, Copyright (c) 2005-2012, by mOo

(tested with and without xcache)

**ownCloud version:** (see ownCloud admin page)
7.0.3 (now updated to 7.0.4 still buggy)

**Updated from an older ownCloud or fresh install:**
updated from 6

**List of activated apps:**
notes
old_menu
bug it still there if i deactivate all apps except share and news

**The content of config/config.php:**
$CONFIG = array (
  'instanceid' => 'ocb75d785c63',
  'passwordsalt' => '',
  'trusted_domains' => 
  array (
    0 => '',
    1 => '',
  ),
  'datadirectory' => '/usr/share/owncloud/data',
  'version' => '7.0.4.2',
  'installed' => true,
  'forcessl' => true,
  'theme' => '',
  'maintenance' => false,
  'loglevel' => '4',
  'log_query' => false,
  'dbtype' => 'mysql',
  'dbname' => 'owncloud',
  'dbhost' => 'localhost',
  'dbtableprefix' => 'oc_',
  'dbuser' => '',
  'dbpassword' => '',
  'overwritewebroot' => '/owncloud',
  'appcodechecker' => false,
  'secret' => '',
);

**Are you using external storage, if yes which one:** local/smb/sftp/...
no

**Are you using encryption:** yes/no
no

### Client configuration
**Browser:**
Firefox
owncloud 1.7.0 mac os X
**Operating system:**
Mac
#### Web server error log
```
Insert your webserver log here
```

#### ownCloud log (data/owncloud.log)
```
Insert your ownCloud log here
```

#### Browser log
```
Insert your browser log here, this could for example include:

a) The javascript console log
b) The network log 
c) ...
```

news team claims it's a core bug : https://github.com/owncloud/news/issues/694

**Comment by PVince81** [comment]
Other bugs were fixed with the same symptoms. Are you still seeing this in 7.0.7 or 8.0.5 ?

**Comment by yopzolo** [comment]
I skipped upgrading until now. I successfully upgraded from 7.0.2 too 8.1.1 and the problem seem solved !
Thank you  ..a lot

**Comment by PVince81** [comment]
@yopzolo great, thanks for the feedback.
From what I remember this must have been fixed in a later 7.0.x version.
